packages/CollabCodeEdit-Server.package/CCEDeployAgent.class/instance/handleUpdate..st

```diff
@@ -0,0 +1,13 @@
+request handling
+handleUpdate: aWebRequest
+	self authenticate: aWebRequest action: [
```

Comment:
This method is very similar to the handleDeploy`. Is it possible to merge those methods?

Reply:
I've tried to extract common logic into parseURL:with:.

Reply:
Looks logic. But I don't understand why you need `tokens` as the parameter if the blocks. They are not used in the first block at least

Reply:
This way, the tokens don't have to be parsed multiple times.

Removing the parameter for one of the blocks would yield an error message along the lines of this: `Error: This block accepts 0 arguments, but was called with 1 argument.`